content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -504,6 +504,22 @@ stops adjusting the target (and sets the `ScalingActive` Condition on itself
 to `false`) until you reactivate it by manually adjusting the target's desired
 replica count or HPA's minimum replica count.
 
+## Recommended Deployment Configuration
+
+When an HPA is enabled, it is recommended that the value of `spec.replcas` of
+the Deployment object is removed.  Without doing so, anytime a change to the
+Deployment object is applied, example via `kubectl apply -f deployment.yaml`,
+will instruct Kubernetes to scale the current number of Pods to the desired
+value retained in the `spec.replica` key.  This may not be desired and could be
```

Comment:
value retained in the `spec.replicas` key.  This may not be desired and could be

Reply:
```suggestion
value retained in the `spec.replicas` key. This may not be desired and could be
```